frontend/app/services/TouchpointBackend.scala

```diff
@@ -107,10 +107,8 @@ object TouchpointBackend {
     )
   }
 
-  val Normal = TouchpointBackend(BackendType.Default)
-  val TestUser = TouchpointBackend(BackendType.Testing)
-
-  val All = Seq(Normal, TestUser)
+  lazy val Normal = TouchpointBackend(BackendType.Default)
```

Comment:
> if you're developing locally and only using UAT, do you really want to insist that DEV is valid?

When developing locally, you're normally using Zuora DEV, so I think you would?


In Membership PROD:
Normal=Zuora PROD
Test = Zuora UAT

In Membership DEV:
Normal=Zuora DEV
Test=Zuora UAT